docs/spark-qualification-tool.md

```diff
@@ -313,95 +347,260 @@ $SPARK_HOME/bin/spark-shell --jars rapids-4-spark-tools_2.12-<version>.jar
+14. Unsupported Write Data Format: Reports the data format which we currently don’t support, i.e.
+    if the result is written in JSON or CSV format.
+15. Complex Types: Looks at the Read Schema and reports if there are any complex types(array, struct or maps) in the schema.
+16. Nested Complex Types: Nested complex types are complex types which
+    contain other complex types (Example: `array<struct<string,string>>`).
+    Note that it can read all the schemas for DataSource V1. The Data Source V2 truncates the schema,
+    so if you see `...`, then the full schema is not available.
+    For such schemas we read until `...` and report if there are any complex types and nested complex types in that.
+17. Potential Problems: Some UDFs and nested complex types. Please keep in mind that the tool is only able to detect certain issues.
+18. Longest SQL Duration: The maximum amount of time spent in a single task of SQL Dataframe operations.
+19. NONSQL Task Duration Plus Overhead: Time duration that does not span any running SQL task.
+20. Unsupported Task Duration: Sum of task durations for any unsupported operators.
+21. Supported SQL DF Task Duration: Sum of task durations that are supported by RAPIDS GPU acceleration.
+22. Task Speedup Factor: The average speed-up of all stages.
+23. App Duration Estimated: True or False indicates if we had to estimate the application duration.
+    If we had to estimate it, the value will be `True` and it means the event log was missing the application finished
+    event, so we will use the last job or sql execution time we find as the end time used to calculate the duration.
+24. Read Schema: Shows the datatypes and read formats. This field is only listed when the argument `--report-read-schema`
+    is passed to the CLI.
+
+**Note:** the Qualification tool won't catch all UDFs, and some of the UDFs can be handled with additional steps.
+Please refer to [Supported Operators](./supported_ops.md) for more details on UDF.
+
+### Stages report
+
+For each step in a physical execution plan, the Qualification tool generates the following information:
+
+1. App ID
+2. Stage ID
+3. Average Speedup Factor: The average estimated speed-up of all the operators in the given stage.
+4. Stage Task Duration: Amount of time spent in tasks of SQL Dataframe operations for the given stage.
+5. Unsupported Task Duration: Sum of task durations for the unsupported operators. For more details,
+   see [Supported Operators](./supported_ops.md).
+6. Stage Estimated: True or False indicates if we had to estimate the stage duration.
+
+Note that this report is currently limited to the CSV file format and not supported in the HTML report.
+
+### Execs report
+
+The Qualification tool generates a report of the "Exec" in each SQL Dataframe operations along with the estimated
+acceleration on the GPU. Please refer to the [Supported Operators](./supported_ops.md) section for more
```

Comment:
it might be nice to add in "_SparkPlan_" or "_Executor Nodes_" to be consistent and that is the seciton they would look in supported operators